config/locales/en/lookup.yml

```diff
@@ -19,6 +19,17 @@ en:
         <p class="govuk-body">Enter the postcode of the place you want to find information about. For example where you live, work, or visit.</p>
         <p class="govuk-body">Each area has a <a class="govuk-link" href="/guidance/local-covid-alert-levels-what-you-need-to-know">Local COVID Alert Level</a>. There are 3 Local COVID Alert Levels. Sometimes these are called ‘tiers’ or known as a ‘local lockdown’.</p>
         <p class="govuk-body">You can find out what you can or cannot do.</p>
+      changes_header: Changes from Saturday 17 October
+      changes_text: The following areas will move into Local COVID Alert Level High,
+      changes_list: |
+        <li>Barrow-in-Furness</li>
+        <li>Chesterfield</li>
+        <li>Elmbridge</li>
+        <li>Erewash</li>
+        <li>Essex</li>
+        <li>London – all boroughs</li>
+        <li>North East Derbyshire</li>
+        <li>York</li>
```

Comment:
It would be better to form the yaml as a list:

```
changes_list:
  - Barrow-in-Furness
  - Chesterfield
  - Elmbridge
  - Erewash
  - etc
```
Rather than partial HTML that could lead to errors.

Reply:
If we did that we'd have to loop through the list the in partial. Probably not worth the time and effort since we need to get this change out.